doc/tests/manual/reports/230725.md

```diff
@@ -0,0 +1,292 @@
+  - Matches on english data fields are ranked above english matches.
+- Actual Result:
+- Test Result: ⚠️
+- Notes: No firs in my results, only spruces.
+
+## TC-007 - Heatmap
+
+- SKIPPED
+
+## TC-008 - Timeline
+
+- Description: Change the date of the map to 'hide' plantings.
+- Preconditions:
+  - User must be on the map planning screen.
+- Test Steps:
+  1. Add a planting to the map.
+  2. Click on the date selection on the bottom of the screen.
+  3. Navigate to a date in the past.
+  4. Wait 1 second.
+- Expected Result:
+  - The indicator was briefly blue, indicating a loading state.
+  - The indicator beside the input is green.
+  - The Date on the bottom/right corner of the screen shows a date in the past.
+  - The planting previously planted is gone.
+- Actual Result:
+- Test Result: ⚠️
+- Notes: Issue [#699](https://github.com/ElektraInitiative/PermaplanT/issues/699), image loading error was present.
+
+## TC-009 - Timeline
+
+- Description: Change the date of the map to 'unhide' plantings.
+- Preconditions:
+  - User must be on the map planning screen.
+- Test Steps:
+  1. Add a planting to the map.
+  2. Click on the date selection on the bottom of the screen.
+  3. Navigate to a date in the past.
+  4. Wait 1 second.
+  5. Navigate to today.
+- Expected Result:
+  - The indicator was briefly blue, indicating a loading state.
+  - The indicator beside the input is green.
+  - The Date on the bottom/right corner of the screen shows the current day.
+  - The planting previously planted was gone while being in the past.
+  - The planting is visible again.
+- Actual Result:
+- Test Result: ⚠️
+- Notes: Issue [#699](https://github.com/ElektraInitiative/PermaplanT/issues/699), image loading error was present.
+
+## TC-010 - Timeline
+
+- Description: Change the `add_date` of a planting to 'hide' it.
+- Preconditions:
+  - User must be on the map planning screen.
+- Test Steps:
+  1. Add a planting to the map.
+  2. Click on the planting.
+  3. Click on the `Add Date` date selector in the left lower toolbar.
+  4. Change the date to a date in the future.
+- Expected Result:
+  - The indicator was briefly blue, indicating a loading state.
+  - The indicator beside the input is green.
+  - The planting previously planted is gone.
+- Actual Result:
+- Test Result: ✔️
+- Notes:
+
+## TC-011 - Timeline
+
+- Description: Change the `remove_date` of a planting to 'hide' it.
+- Preconditions:
+  - User must be on the map planning screen.
+- Test Steps:
+  1. Add a planting to the map.
+  2. Click on the planting.
+  3. Click on the `Remove Date` date selector in the left lower toolbar.
+  4. Change the date to today.
+- Expected Result:
+  - The indicator was briefly blue, indicating a loading state.
+  - The indicator beside the input is green.
+  - The planting previously planted is gone.
+- Actual Result: Error message: "Das Datum an dem die Pflanze hinzugefügt wurde muss vor dem Datum liegen, an dem sie entfernt wurde."
+- Test Result: ❌
```

Comment:
Yes, the protocol seems to be incomplete, you first need to change the "added date" to something earlier.